src/ts/ConvivaAnalytics.ts

```diff
@@ -655,12 +650,20 @@ export class ConvivaAnalytics {
     this.onPlaybackStateChanged(event);
   };
 
+  private onTimeChanged = (event: TimeChangedEvent) => {
+    if (!this.isSessionActive()) {
+      return;
+    }
+
+    this.playheadTimeMs = event.time * 1000;
```

Comment:
Does Conviva provide any guidance on the expected value for Live streams? Asking because `event.time` will be the absolute epoch time in live streams. We could also provide the relative time if that is required (`player.getCurrentTime('relativetime')`)

Reply:
My understanding is Conviva expects relative time. This way seems better than using event.time so I'll switch to it. Does player 8.6.0 support (player.getCurrentTime('relativetime'))? Because the player api typings file is telling me it doesn't expect any arguments. Or does this always return relative time without an argument? Even for live?

Reply:
No, 8.6.0 doesn't support this, it was added in [8.31.0](https://bitmovin.com/docs/player/releases/web/web-8-31-0). We may want to raise the minimum player version.